SPECS/sos/sos.spec

```diff
@@ -1,25 +1,28 @@
 %{!?python3_sitelib: %global python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 Summary:        A set of tools to gather troubleshooting information from a system
 Name:           sos
-Version:        4.4
-Release:        2%{?dist}
-License:        GPLv2+
+Version:        4.6.1
+Release:        1%{?dist}
+License:        GPL-2.0-or-later
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/sosreport/sos
 #Source0:       https://github.com/sosreport/sos/archive/%%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
-BuildRequires:  gettext
+# The sos-4.6.1.tar.gz is missing a commit to bump the version to 4.6.1
+# https://github.com/orgs/sosreport/discussions/3492
+Patch0:         bump-version-4-6-1.patch
 BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
 Requires:       bzip2
 Requires:       python3
 Requires:       python3-libxml2
-Requires:       python3-magic
 Requires:       python3-pexpect
 Requires:       python3-rpm
-Requires:       tar
-Requires:       xz
+Requires:       python3-setuptools
+Recommends:     python3-magic
```

Comment:
I agree, I was on the fence about this change. For most of the changes here I was trying to follow the upstream changes. I will use requires

Reply:
ah ok sorry i didn't realize the changes were coming from upstream, so looks like they did this because of https://github.com/sosreport/sos/issues/3021 which isn't a problem for us (since we do have python3-magic) so yeah, it probably does make sense to keep it Requires:

Reply:
unresolving this just as fyi so you can see my comment

Reply:
Sounds good, I see it :) 

Resolving this.